### PR TITLE
Set layout master

### PR DESCRIPTION
**Pull request by at-minhdao**
## Description
- Set layout master
- Create route /admin for dashboard
## Reference Requirements
- Product Backlog ID: PBI-2
- Task ID: 2.3

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Non SQL

### CODE REVIEW

File: resources/views/layout/admin/app.blade.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@include('layout.admin.header')
+@include('layout.admin.side-bar')
+@yield('content')
+@include('layout.admin.footer')
```
Comment (at-hieule) [inline resources/views/layout/admin/app.blade.php]: Check convention nhé!

File: resources/views/layout/admin/header.blade.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                </div>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <!-- Left side column. contains the logo and sidebar -->
+
```
Comment (at-hieule) [inline resources/views/layout/admin/header.blade.php]: Convention nhé!

File: resources/views/layout/admin/app.blade.php (hunk 1 above)
Comment (at-hieule) [inline resources/views/layout/admin/app.blade.php]: Theo mình file app.blade.php sẽ chứa đường dẫn của css và script. Dùng @yield như sau : 
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  
  <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
  <link href="{{ URL::asset("bower_components/bootstrap/dist/css/bootstrap.min.css") }}" rel="stylesheet">
 ......
 @yield('title_page')
</head>
<body class="hold-transition skin-blue sidebar-mini">
  @include('layouts.admin.header')
  @include('layouts.admin.menu')

  <div class="wrapper">
  @yield('content')
  </div>
  @include('layouts.admin.footer')
  <script src="{{ URL::asset("bower_components/jquery/dist/jquery.min.js") }}"></script>
 ....
  @yield('script')
</body>
</html>

File: resources/views/layout/admin/app.blade.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>AdminLTE 2 | Dashboard</title>
```
Comment (at-hieule) [inline resources/views/layout/admin/app.blade.php]: Theo mình mỗi trang có title riêng. Dùng @yield('title_page')

**Comment by at-hieule** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**